spec/models/delay_henka/scheduled_change_spec.rb

```diff
@@ -3,39 +3,24 @@
 module DelayHenka
   RSpec.describe ScheduledChange, type: :model do
 
-
-    describe '.cleanup model cast' do
```

Comment:
The specs are organized by methods. `describe '.schedule'` tests against the class method `.schedule`; `describe '#apply_change'` tests against the instance method `#apply_change`.

Reply:
@zinosama ok, at before, also try to place the tests at `.schedule`， but not convenient to custom the `let(:changeable)` at those 2 tests, so choose extract it.